account_tax_group_widget_base_amount/static/src/js/tax_group.js

```diff
@@ -0,0 +1,35 @@
+odoo.define("account_tax_group_widget_base_amount.tax_group", function(require) {
+    "use strict";
+
+    var core = require("web.core");
+    var QWeb = core.qweb;
+    var fieldRegistry = require("web.field_registry");
+    var TaxGroupCustomField = fieldRegistry.get("tax-group-custom-field");
+
+    TaxGroupCustomField.include({
+        /**
+         * This method checks that there are several taxes in the document thus
+         * the base should be showed or not.
+         *
+         * @returns boolean (true if )
+         */
+        _isMultiTax: function() {
+            return this.value && this.value.length > 1;
+        },
+        _render: function() {
+            var _this = this;
+            if (this._isMultiTax()) {
+                this.$el.html(
+                    $(
+                        QWeb.render("AccountTaxGroupTemplate", {
+                            lines: _this.value,
+                            displayBase: true,
```

Comment:
What about `displayEditWidget: displayEditWidget,` that I found in core?

Reply:
I've redone it to avoid js override